pulp_container/app/tasks/sync_stages.py

```diff
@@ -115,10 +117,13 @@ async def run(self):
         async with ProgressReport(
             message="Downloading tag list", code="sync.downloading.tag_list", total=1
         ) as pb:
-            repo_name = self.remote.namespaced_upstream_name
+            try:
+                repo_name = filter_repo(self.remote)
+            except InvalidRequest:
```

Comment:
A side note: this is a `rest_framework` exception that is supposed to be used within the api-app stack. It helps us to better render error messages. It is not very helpful to "raise" an HTTP 400 error from a task running inside a worker because there is no HTTP interface involved and a user cannot benefit from it.